man/systemd.exec.xml

```diff
@@ -2916,6 +2916,29 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LogFilterPatterns=</varname></term>
+
+        <listitem><para>Define a PCRE regular expression to filter log messages based on the MESSAGE= field of the
+        structured message. If the first character of the pattern is <literal>~</literal>, log messages
+        matching the pattern should be discarded. This option takes a single pattern as an argument but can
+        be used multiple times to create a list of allowed and denied patterns. If the empty string is
+        assigned, the filter is reset, and all prior assignments will have no effect.</para>
```

Comment:
maybe add a comment here explaining how to do a regex that matches a literal `~`. Because if you do `~foobar` we'd of course understand that as "exclude 'foobar'", but if people want to say "include '~foobar'" we have to tell them how.

iiuc, then pcre accepts `\x` escapes, so we can probably simply suggest people to write `\7xefoobar` if they want to include '~foobar'.

Maybe include such a string in the test case actually